File: AD-HOC/2466/main.rb
# frozen_string_literal:true

def last_color(colors)
  colors = _next_row(colors) while colors.length > 1
  colors.last
end

def _next_row(colors)
  next_row = []
  (colors.length - 1).times do |i|
    next_row.push(colors[i] == colors[i + 1] ? 1 : -1)
  end
  next_row
end

gets # ignore first line
colors = gets.split.map(&:to_i)
puts last_color(colors).positive? ? 'preta' : 'branca'

File: AD-HOC/2466/main.py
def last_color(colors):
    while len(colors) > 1:
        colors = _next_row(colors)
    return colors[0]


def _next_row(colors):
    next_row = []
    for i in range(len(colors)-1):
        next_row.append(1 if colors[i] == colors[i+1] else -1)
    return next_row


input() # ignore first line
colors = list(map(int, input().split()))
print('preta' if last_color(colors) > 0 else 'branca')
